api/src/main/resources/workbench.yaml

```diff
@@ -2159,10 +2175,58 @@ paths:
           schema:
             $ref: "#/definitions/DataSetPreviewResponse"
 
+  /v1/offline/syncResearcherData:
+    post:
+      tags:
+        - offlineRDRExport
+      description: >
+        Send Researcher data
+      operationId: "syncResearcherData"
+      parameters:
+        - in: body
+          name: researchIds
+          required: true
+          description: The IDs of concepts to be added to the concept set.
+          schema:
+            type: object
```

Comment:
Is there a reason this needs to be an object rather than an array of numbers? You could follow the pattern from CreateConceptSetRequest:

        type: array
        description: The IDs of concepts to be added to the concept set.
        items:
          type: integer
          format: int64

Reply:
I will be updating the comment, since this is not integer but a comma separated list of ids, since we can send up task body can just contain ByteStrings